go.mod

```diff
@@ -1,17 +1,31 @@
 module github.com/iver-wharf/wharf-provider-github
 
-go 1.13
+go 1.16
 
 require (
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
-	github.com/gin-contrib/cors v1.3.0
-	github.com/gin-gonic/gin v1.4.0
+	github.com/gin-contrib/cors v1.3.1
+	github.com/gin-gonic/gin v1.7.2
+	github.com/go-openapi/spec v0.20.3 // indirect
```

Comment:
Suggest running `go mod tidy` as some of these can be cleaned up. Especially the `go.sum` file

Reply:
Ran go mod tidy and added the changes as fixups. Does not seem to have made that big a difference.

Reply:
If it didn't have much difference, then those dependencies really need to be there. /shrug